packages/erc3k/contracts/IERC3000.sol

```diff
@@ -8,20 +8,25 @@ pragma experimental ABIEncoderV2;
 import "./ERC3000Data.sol";
 import "./IERC3000Executor.sol";
 
+// Note: why not defining functions as external?
 abstract contract IERC3000 {
     function schedule(ERC3000Data.Container memory container) virtual public returns (bytes32 actionHash);
+    // Note: why logging just the collateral? wouldn't be better to log a Config hash or identifier?
```

Comment:
We could remove collateral tbh since it should be known since it is in the config